lib/auth/auth_with_roles_test.go

```diff
@@ -373,12 +373,14 @@ func TestAPILockedOut(t *testing.T) {
 	})
 	require.NoError(t, err)
 	require.NoError(t, srv.Auth().UpsertLock(ctx, lock))
+	require.Eventually(t, func() bool { return testOp() != nil }, time.Second, time.Second/10)
 	err = testOp()
 	require.Error(t, err)
```

Comment:
You can probably remove this check, since the eventually picked it up, right?

(same goes for the checks on lines 384 and 394.